### DISCUSSION TITLE
Multi Line Text Option in Attribute

### DISCUSSION THREAD

**Discussion by chrisBradley-dev** [unopim/unopim#71] [Ideas]
Would be really useful if you could have an multiline text object, which would align with shopify multi_line_text_field and other array based text objects in other channels.

https://shopify.dev/docs/apps/build/custom-data/metafields/list-of-data-types

**Comment by navneetkumar-pim-webkul** [comment]
@chris-chameleonbrands 

Thank you for your feedback! Adding a multiline text object to align with Shopify's multi_line_text_field and similar array-based text objects in other channels is a great suggestion. We'll analyze its feasibility and assess how it can fit into our roadmap.
